### ISSUE TITLE
staticOptions do not works?

### ISSUE THREAD

**Issue by littlee** [webpack/webpack-dev-server#490]
my server.js
```js
var webpack = require('webpack');
var WebpackDevServer = require('webpack-dev-server');
var config = require('./webpack.dev.js');

var PORT = 3333;

new WebpackDevServer(webpack(config), {
	publicPath: '/__build__/',
	hot: true,
	historyApiFallback: true,
	stats: {
		colors: true
	},
	staticOptions: {
		index: './dev.html'
	}
}).listen(PORT, 'localhost', function(err) {
	if (err) {
		console.log(err);
		return;
	}
	console.log(':::Server Running::: ==> localhost:' + PORT);
});
```

I want to serve "dev.html" instead of "index.html", but it does not works.
![qq 20160526101052](https://cloud.githubusercontent.com/assets/7534668/15561599/237fc7ba-232a-11e6-83f8-225bc0910f4e.png)

**Comment by faddee** [comment]
What version of webpack-dev-server do you have? `staticOptions` is only available from `v2.1.0-beta.0` as you can see in https://github.com/webpack/webpack-dev-server/commit/e5f698deabbb8a4b6d213b43098db85880d62ba4.

**Comment by littlee** [comment]
well, my version is 1.14.1 :flushed:
